packages/core/src/components/ChoiceList/Choice.scss

```diff
@@ -161,6 +187,17 @@ Style guide: components.choice
   border-radius: 100%;
 }
 
+// Display label on the left side of the checkbox
+.ds-c-choice--left-label + label {
```

Comment:
Thoughts on changing this modifier to `ds-c-choice--right`? Since the class is applied to the input, rather than the label, I think this still makes sense and is inline with other modifiers like `ds-u-float--right` and `ds-u-text-align--right`.

Reply:
Yeah I think that's reasonable. I wasn't entirely sure what to call it, so I figured we'd discuss it in a review.